jekyll/_cci2/skip-build.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: classic-docs
+title: Skipping a Build
+---
+
+By default,
+CircleCI automatically builds a project
+whenever you push changes to a version control system (VCS).
+You can override this behavior
+by adding a `[ci skip]` or `[skip ci]` tag
+anywhere in a commit's title or description.
+This not only skips the marked commit,
+but also **all other commits** in the push.
+
+If you later decide to build a skipped commit,
+click the _Rebuild_ button on the build details page in CircleCI.
```

Comment:
Rebuild is different now, so we should add a sentence about that and modify the button name. See https://circleci.com/changelog/#rerun-button-change-on-the-jobs-page